etcdserver/api/v2store/node_test.go

```diff
@@ -239,7 +239,7 @@ func newTestNode() *node {
 
 func newTestNodeDir() (*node, *node) {
 	s := newStore()
-	nd := newDir(s, key, 0, nil, time.Now().Add(expiration))
+	nd := newDir(s, key, 0, s.Root, time.Now().Add(expiration))
 	cKey, cVal := "hello", "world"
 	child := newKV(s, cKey, cVal, 0, nd, time.Now().Add(expiration))
```

Comment:
do we use this value in current tests or is the plan to use this in future tests?

Reply:
Currently there is no. i will close it.